### DISCUSSION TITLE
October 2024 Conflux Documentation Contributor Incentives

### DISCUSSION THREAD

**Discussion by darwintree** [Conflux-Chain/conflux-documentation#793] [Announcements]
Here is the eligibility list of contributors who will receive the contributor incentives airdrop for 202410. We deeply appreciate your support and highly value the insights and expertise you bring to our community.

以下为202410有资格获取激励的贡献者名单。非常感谢名单中的各位对本项目的支持。

The following GitHub contributors will receive the airdrop:

以下的GitHub贡献者将获得空投：

* [0xffffb](https://github.com/0xffffb)

* [jackleeio](https://github.com/jackleeio)

* [Vikash-8090-Yadav](https://github.com/Vikash-8090-Yadav)

The following Crowdin contributors will receive the airdrop:

以下的Crowdin贡献者将获得空投：

* [0xffffb](https://github.com/0xffffb)

* [PingJing](https://crowdin.com/profile/PingJing)

The eligibility list is curated in accordance with our [CONTRIBUTING guideline](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING). If you believe your contribution has been overlooked, please do not hesitate to contact us.

名单是根据[CONTRIBUTING指导原则](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING)所挑选的。如果您有对应贡献但被遗漏在了名单外，欢迎联系我们。

## Contributor Incentives Details 贡献者激励详情

At present, we offer corresponding incentives for Chinese translators. Translator incentives are calculated based on the volume of translations and whether the translation is accepted. We will airdrop rewards to translators who have provided their addresses. For the method of providing addresses, please refer to [Conflux Technical Documentation Translation Plan (2024.03.06) ](https://forum.conflux.fun/t/conflux-2024-03-06/20286).

目前我们为中文翻译者提供相应激励，翻译者激励计算与翻译量与翻译是否被接受相关。我们将为已提供了地址的翻译者空投激励。地址提供方式请参考[Conflux 技术文档翻译计划（2024.03.06）](https://forum.conflux.fun/t/conflux-2024-03-06/20286)。

We have introduced financial incentives for contributors which are directly related to the quality of the contributor's work.

我们为仓库的贡献者引入的经济激励是与贡献者的工作质量所直接相关的。

For incentives for documentation content contributors, please refer to:

https://www.notion.so/confluxglobal/cfe6a4c30bf4473f83a83e5efe9a86c9?v=138867639969810da062000c744cdfec

and their incentives are calculated according to the [2024 reward guidelines](https://github.com/Conflux-Chain/conflux-documentation/blob/2024Q1-reward/docs/general/CONTRIBUTING/reward-guidelines.md). If you have relevant contributions but have been omitted from the list, please feel free to contact us. In addition, we will collect the (temporary) addresses provided by the relevant contributors for airdrop redemption in this discussion.

文档内容贡献者的激励请参考：

https://www.notion.so/confluxglobal/cfe6a4c30bf4473f83a83e5efe9a86c9?v=138867639969810da062000c744cdfec

其激励根据[2024 reward guidelines](https://github.com/Conflux-Chain/conflux-documentation/blob/2024Q1-reward/docs/general/CONTRIBUTING/reward-guidelines.md)计算。如果您有对应贡献但被遗漏在了名单外，欢迎联系我们。此外我们将在本discussion中收集相关贡献者提供的领取空投的（临时）地址。

Thank you for your assistance in enhancing our project. If you have any queries or require assistance, please feel free to create an issue in our Github repository.

感谢各位持续帮助我们改进项目。如果你还有任何疑问或需要帮助，请随时在我们的Github仓库中创建issue。

**Comment by darwintree** [comment]
@Vikash-8090-Yadav  Thank you for your contribution! Would know your Conflux core space's address for the airdrop!

**Reply by Vikash-8090-Yadav** [reply]
Thanks for the mention, Yes my address is:

core: cfx:aamth1028tuuug2tva37z2egae0cxcjt3jyjy7f1th

espace: 0x7D4b4ffC9A1A3D0113fFbc4e169820c94fe1295b

**Reply by Vikash-8090-Yadav** [reply]
Hey @darwintree Gm GM 
  I updated my address. Please consider these ones. Thanks a lot again!

**Reply by Vikash-8090-Yadav** [reply]
Hey @darwintree when can we expect the payout?

**Reply by darwintree** [reply]
> Hey @darwintree when can we expect the payout?

There might be a one-month latency for developers who first contribute to our repo,but would be then within 1 or 2 week after that due to our internal process. Sorry for not clearly stating the time.
